include/solvers/vof.h

```diff
@@ -65,7 +65,7 @@ class VolumeOfFluid
     , triangulation(p_triangulation)
     , simulation_control(p_simulation_control)
     , dof_handler(*triangulation)
-    , pfg_dof_handler(*triangulation)
+    , filtered_phase_fraction_gradient_dof_handler(*triangulation)
```

Comment:
this is more verbose but really clearer! Would you have an idea that would be both clear and a bit more concise? If not, clarity is better

Reply:
well, it was Bruno's comment. if you have any suggestions, I can use it. :)

Reply:
Does `pfg` stand for `phase_fraction_gradient`? If yes, why did you specify `filtered` after? I also wonder whether you really need the `fraction` part: as we only have two phase values (0 and 1), the phase value of a given node directly gives the fraction of both fluids at this location. In all, could this be only `phase_grad_dof_handler`? Do not hesitate to tell me if I missed something!

Reply:
it is filtered phase fraction gradient because the weak form of the equation in the assembler is defined to remove oscillations from the phase fraction gradient. It is fraction because in the interface we have phase fraction values between 0 and 1, and this parameters calculates the gradient of these phase fractions. I know the name is verbose, but I think it's better to keep it because it is descriptive.

Reply:
Alright, this is your call, I just gave you suggestions as you asked :)
Please just make sure that `pfg` is replaced everywhere